### PR TITLE
Cart component - "openCheckout" event - not firing

### PR DESCRIPTION
**Pull request by hamisor**
I have added the missing user-event 'openCheckout' to the cart component.

### CODE REVIEW

**Comment by ChalkyBrush** [comment]
would like this event to work as well

**Comment by danielcolgan** [comment]
@tessalt what is issue with merging this PR ?

**Comment by tombennet** [comment]
@spencercanner Are there any issues with merging this PR? 

I've tested the bug fix in a newly-forked repo and it restores expected functionality (i.e. the checkout event fires). See eeaec6c.

Thanks!

File: src/components/cart.js [hunk 1]
```diff
@@ -190,7 +190,8 @@ export default class Cart extends Component {
   }
 
   onCheckout() {
-    this.checkout.open(this.model.checkoutUrl);
+	  this._userEvent('openCheckout');
```
Comment (spencercanner) [inline src/components/cart.js]: For consistency, could you indent with 2 spaces instead of a tab?

Comment (hamisor) [reply src/components/cart.js]: hi @spencercanner, sorry for taking so long to get back to you. I saw @tombennet has a new PR open to address the coding style issue and I'm happy to close this PR. Please let me know if you need any help. Thanks.

**Comment by melissaluu** [comment]
Thanks @hamisor - I'll close this PR since #647 is merged.

**Pull request closed without merging**